docs/resources/aws_ebs_volumes.erb

```diff
@@ -0,0 +1,89 @@
+---
+title: About the aws_ebs_volumes Resource
+platform: aws
+---
+
+# aws\_ebs\_volumes
+
+Use the `aws_ebs_volumes` InSpec audit resource to test properties of some or all AWS EBS volumes. To audit a single EBS volume, use `aws_ebs_volume` (singular).
+
+EBS volumes are persistent block storage volumes for use with Amazon EC2 instances in the AWS Cloud.
+
+Each EBS volume is uniquely identified by its ID.
+
+<br>
+
+## Availability
+
+### Installation
+
+This resource is distributed along with InSpec itself. You can use it automatically.
+
+
+## Syntax
+
+An `aws_ebs_volumes` resource block collects a group of EBS volumes and then tests that group.
+
+    # Ensure you have exactly 3 volumes
+    describe aws_ebs_volumes do
+      its('volume_ids.count') { should cmp 3 }
+    end
+
+    # Use the InSpec resource to enumerate IDs, then test in-depth using `aws_ebs_volume`.
+    aws_ebs_volumes.volume_ids.each do |volume_id|
+      describe aws_ebs_volume(volume_id) do
+        it { should exist }
+        it { should be_encrypted }
+        its('size') { should = 8}
+        its('iops') {should cmp 100}
+      end
+    end
+
+<br>
+
+## Examples
+
+As this is the initial release of `aws_ebs_volumes`, its limited functionality precludes examples.
+
+<br>
+
+## Filter Criteria
+
+This resource currently does not support any filter criteria; it will always fetch all volumes in the region.
+
+## Properties
+
+### entries
+
+Provides access to the raw results of the query, which can be treated as an array of hashes. This can be useful for checking counts and other advanced operations.
+
+    # Allow at most 100 EBS volumes on the account
+    describe aws_ebs_volumes do
+      its('entries.count') { should be <= 100}
+    end
+
+
+### volume_ids
+
+Provides a list of the volume ids that were found in the query.
+
+    describe aws_ebs_volumes do
+      its('volume_ids') { should include('i-12345678') }
```

Comment:
Not sure if it's documented anywhere (working on a style guide) but we usually do:

```
include 'whatever'
```

Reply:
Can you provide more detail or a link to a correct example? I'm not following your request.

Reply:
Sure, again, style issue and nothing functional:

```
its('volume_ids') { should include 'i-12345678' }
```

This may seem counter intuitive because we require the opposite in our codebase, but we try to keep code the InSpec user writes as human readable as possible (i.e. able to be read/written by people who aren't developers')

Reply:
Disregard. I see what you are saying now